src/App.tsx

```diff
@@ -263,7 +267,22 @@ export const App = ({
 		) {
 			onExpanded(window.performance.now() - expandedStartTime);
 		}
-	}, [isExpanded]);
+
+		if (isExpanded) {
```

Comment:
Shall we break this out into it's own `useEffect`? The `onExpanded` timer is something we want to fire at the same point but it's enough of a separate logical action to keep it apart I would say

Reply:
Aye, totally, I had that originally but got your feeling of "There's definately a lot of useEffects" but makes more sense seperate.